content/docs/projects/approver-policy.md

```diff
@@ -72,13 +72,11 @@ selector.
 least one policy is appropriate for the request but none of those permit the
 request, the request is denied.**
 
-A denied CertificateRequest is considered to be permanently failed. If it was
-created for a Certificate resource, the issuance will be retried with
-[exponential
-backoff](../faq/README.md#what-happens-if-issuance-fails-will-it-be-retried)
-like all other permanent issuance failures. A CertificateRequest that is neither
-approved nor denied (because no matching policy was found) will not be further
-processed by cert-manager until it gets either approved or denied.
+A denied CertificateRequest is considered to be permanently failed. Currently if
+a CertificateRequest that was created for a Certificate resource gets denied,
+issuance might or might not be retried with [exponential
+backoff](../faq/README.md#what-happens-if-issuance-fails-will-it-be-retried) (a
+new CertificateRequest created after a delay).
```

Comment:
Maybe add a bit more detail about when it might and when it might not get retried.
I can't remember what we found out.

Reply:
Currently it depends on which controller comes first in the race condition. However, I will update this PR and point it at next docs since https://github.com/cert-manager/cert-manager/pull/5887 got merged that makes the behaviour predictable.